src/components/Popover/README.md

```diff
@@ -284,6 +284,39 @@ React.createElement(() => {
 });
 ```
 
+## Controlled Popover
+
+To make it easier to implement the popover in your app, you can set the
```

Comment:
I would mention, that the clean solution would be `anchor` API, but that it is not available. Maybe a link to MDN.

Reply:
I would consider enhancing the documentation to something like:

```
Popover API can be used to control visibility of Popover component. You need to set `id` on the trigger element and  matching `popoverHelperId` attribute on the Popover component.  This leverages the browser's Popover API to control the popover, automatically closing it when the trigger or the backdrop is pressed. 
```

I would explicitly mention setting `id` on the target element so it is crystal clear.